### ISSUE TITLE
accessability: hint buttons

### ISSUE THREAD

**Issue by alekssamos** [docker/for-win#12486]
1. Run and install [NVDA](https://github.com/nvaccess/nvda/);
2. Install and run Docker Desktop;
3. Go to the Docker Dashboard (which is made as a Chrome HTML document).
The Start, stop, delete, menu and other buttons themselves do not have any sayings. NVDA says just the word "button". But the hints appear separately from the buttons when focusing on them and disappear when the focus goes away. Accordingly, the screen reader reads buttons separately, prompts separately. At the same time, it is difficult to catch the keyboard focus and put it on the button I need. Which button which hint refers to is also difficult to understand.
4. Not all buttons have these hints. For example, they are not in the images section to open the context menu.

Solving the problem:
Use `aria-label` attributes for screen readers. Display the existing hints there and add them to the places where they are missing.
